post_installation_configuration/cluster-tasks.adoc

```diff
@@ -94,3 +94,31 @@ Understand and configure Pod disruption budgets.
 
 include::modules/nodes-pods-pod-disruption-about.adoc[leveloffset=+2]
 include::modules/nodes-pods-pod-disruption-configuring.adoc[leveloffset=+2]
+
+[id="post-install-rotate-cloud-creds"]
+== Rotating cloud provider credentials
+Some organizations require rotating the cloud provider credentials that were used to install {product-title} after installation. The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)] manages cloud provider credentials and can be configured to operate in several different modes. The process for rotating cloud credentials depends on the mode the CCO is configured to use.
+
+[id="cred-rotation-cco-mode-mint"]
+=== Rotating credentials with the CCO in mint mode
+When the CCO is configured to operate in _mint mode_, it uses the provided admin-level cloud credential to create new credentials for each component with only the specific permissions that are required. Rotating credentials in mint mode requires manual removal of component credentials that were created by the removed credential.
+
+. Update the admin credentials root secret for your cloud provider.
+
+. Look up every credential that has been minted for the `credentialsRequest` CR and delete it from the cloud provider.
+
+. Delete each component secret.
+//Is this on the cluster? Do they need a bit more info to know what they're looking to delete and/or how to do so?
+
+. Provide the new admin-level cloud credential.
+
+The CCO uses the new admin-level cloud credential to mint new credentials for each component.
+
+[id="cred-rotation-cco-mode-passthrough"]
+=== Rotating credentials with the CCO in passthrough mode
+When the CCO is configured to operate in _passthrough mode_, it passes the provided cloud credential to the components that request cloud credentials. The credential must have permissions to perform the installation and complete the operations that are required by components in the cluster, but does not need to be able to create new credentials.
+
+After rotating the passthrough mode cloud credential, it can take up to 11 hours to propagate the new credential to all of the components that need it. During this time, the cluster might not be fully operational. To reduce the time to propagate the new credential, delete the CCO pod. Deleting the CCO pod causes the CCO to immediately propagate the new credential, which can take up to an hour to complete.
```

Comment:
@akhil-rane is this true? I thought your PR to handle passthrough credentials rotation will react to a brand new passthrough secret to trigger updating each Secret as needed...

Reply:
If so, lmk which 4.5.z it went into because we should call that out

Reply:
This is likely not true anymore for 4.6, but the fix does not look like it was backported to 5.4.

Reply:
I lied, it was 4.7 this went in. @mjudeikis requests it be backported to 4.6.

Reply:
Will leave this step in for 4.5-4.6, and open a new PR for master/4.7 omitting it. Can cherrypick down into z-streams as Akil's fix makes its way down into them.